docs/en/ingest-management/integrations/view-integration-policies.asciidoc

```diff
@@ -5,7 +5,9 @@
 <titleabbrev>View integration policies</titleabbrev>
 ++++
 
-An integration policy is created when you add an integration to an {agent}
+:integrations_link: https://www.elastic.co/integrations/data-integrations
+
```

Comment:
```suggestion
```

Reply:
Good idea, but let's point to the integrations documentation rather than the marketing page (I think the marketing page still has some links to Beats modules, which might be confusing to users.) Note that we already have a [shared attribute](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc) defined for that book.